### PR TITLE
Term Entry - PyTorch - nn - nn.Dropout()

### LINKED ISSUE #5232: [Term Entry] PyTorch nn nn.dropout()
### Reason/inspiration (optional)

We would like a new entry on the `nn.dropout()` in PyTorch. The entry should go in a new file under `content/pytorch/concepts/nn/terms/dropout/dropout.md`.      _x000D_
  _x000D_
The entry should include:  _x000D_
  _x000D_
- An introduction to the concept     _x000D_
  _x000D_
- A `Syntax` section that provides the syntax for the concept (if applicable)     _x000D_
  _x000D_
- An `Example` section that provides an example demonstrating the concept in use (if applicable)     _x000D_
  _x000D_
Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by danitellini**
### Description

Topic: PyTorch `nn.Dropout()`
Files added: `docs/content/pytorch/concepts/nn/terms/dropout/dropout.md`

Completed to close open issue

### Issue Solved

Closes #5232 

### Type of Change

- Adding a new entry
- ~Editing an existing entry (fixing a typo, bug, issues, etc)~
- ~Updating the documentation~

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by danitellini** [comment]
I will be editing this one more time based on a review on a similar topic

File: content/pytorch/concepts/nn/terms/dropout/dropout.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  - 'Machine Learning'
+  - 'Python'
+Tags:
+  - 'Deep Learning'
+  - 'Machine Learning'
+  - 'Neural Networks'
+  - 'PyTorch'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/machine-learning'
```
Comment (mamtawardhani) [inline content/pytorch/concepts/nn/terms/dropout/dropout.md:14]:
```suggestion
Title: 'nn.dropout()'
Description: 'Returns a tensor with the same shape as the input, where some elements are randomly set to zero based on the dropout probability to prevent overfitting.'
Subjects:
  - 'AI'
  - 'Machine Learning'
Tags:
  - 'Deep Learning'
  - 'Machine Learning'
  - 'Neural Networks'
  - 'PyTorch'
CatalogContent:
  - 'learn-python-3'
  - 'paths/machine-learning'
```

File: content/pytorch/concepts/nn/terms/dropout/dropout.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+**`torch.nn.Dropout`** is a regularization technique in [PyTorch](https://www.codecademy.com/resources/docs/pytorch) used to prevent overfitting during [neural network](https://www.codecademy.com/resources/docs/pytorch/nn) training. It works by randomly setting a fraction of input units to zero during each forward pass, reducing the model's reliance on specific neurons and encouraging it to generalize better.
+
+Dropout is only applied during training and is automatically disabled during evaluation.
+
+## Syntax
+
+The syntax for applying dropout in PyTorch is:
+
+```pseudo
+torch.nn.Dropout(p=0.5)
```
Comment (mamtawardhani) [inline content/pytorch/concepts/nn/terms/dropout/dropout.md:26]:
```suggestion
torch.nn.Dropout(p=0.5, inplace=False)
```

File: content/pytorch/concepts/nn/terms/dropout/dropout.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+```
+
+- `p`: The probability of dropping of a unit (default is `0.5`, meaning 50% dropout).
+  - A higher value increases regularization but may reduce model capacity.
+- Returns a module that applies dropout to the input [tensor](https://www.codecademy.com/resources/docs/pytorch/tensors).
```
Comment (mamtawardhani) [inline content/pytorch/concepts/nn/terms/dropout/dropout.md:31]: Can you please add the `inplace` parameter description as well, and put the return statement outside the bullet points?

File: content/pytorch/concepts/nn/terms/dropout/dropout.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+Input Tensor:
+tensor([[ 0.3369, -1.0319,  1.3535, -0.4457, -0.2493],
+        [ 0.3731,  1.0746, -0.5612, -1.2638, -0.0648],
+        [ 0.9374, -0.2997, -1.1414,  0.4212,  0.7359]])
+
+Output Tensor with Dropout Applied:
+tensor([[ 0.6738, -0.0000,  2.7070, -0.8914, -0.4986],
+        [ 0.7462,  2.1492, -0.0000, -2.5275, -0.1297],
+        [ 0.0000, -0.5994, -2.2827,  0.8423,  1.4718]])
+```
```
Comment (mamtawardhani) [inline content/pytorch/concepts/nn/terms/dropout/dropout.md:68]: At the end, a note can be added as follows:

> **Note**: The output will change each time because dropout randomly sets a fraction of the input values to zero during each forward pass, introducing variability in the results.

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @danitellini thank you for contributing to Codecademy Docs, the entry is nicely written! 😄 

I've suggested a few modifications, could you please review and make those changes at your earliest convenience? Thank you! 😃

**Comment by mamtawardhani** [comment]
Let me know once you make the changes @danitellini 😄

**Review by mamtawardhani — APPROVED** [review]
Thank you for making the required changes @danitellini 😄 

The entry looks good for a second round of review! 🚀

**Review by Sriparno08 — APPROVED** [review]
Looks good, @danitellini!
